compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -152,6 +152,11 @@ void GlobalOptimizationOptions::bindOptions(OptionsBinder &binder) {
                    llvm::cl::desc("Strips debug assertions after any useful "
                                   "information has been extracted."),
                    llvm::cl::cat(category));
+  binder.opt<std::string>(
```

Comment:
this can be a repeated flag so that multiple pattern modules can be provided - then one pass can add all the patterns and run them all at once